docker/kubernetes-tentacle/scripts/configure-tentacle.sh

```diff
@@ -115,9 +115,6 @@ function configureTentacle() {
         tentacle configure --instance "$instanceName" --port $internalListeningPort --noListen "False"
     fi
 
-    echo "Updating trust ..."
-    tentacle configure --instance "$instanceName" --reset-trust
```

Comment:
This clears out the `TrustedOctopusServers` which is set when registering. If we skip registering when we've already registered, we effectively clear out the trusted servers and then never replace them meaning a restart of the container fails.

I've remove this because we set the trusted server once and then never clear it. Is this going to cause us issues?

Reply:
Rather than removing it, could we skip some of this configuration if we know it's configured? Or do we need to set some information before we start the tentacle app?

Reply:
If we just skip it when "isRegistered" is true, the only time it would be executed is when "isRegistered" is false, which is when there would be no trusted servers configured anyway. So I thought it didn't really make sense to have it when running in kubernetes...